scripting/get5/stats.sp

```diff
@@ -1137,47 +1139,52 @@ static void PrintDamageInfo(int client) {
     return;
 
   char message[256];
+  int msgSize = sizeof(message);
```

Comment:
I know that the whole `sizeof(message)` gets to be repeated a lot, but this is the first time it appears in any of the codebase. Either we refactor everywhere (could be done in separate commit), or we change styles to have this for every `sizeof` call that's used more than once in a function call?